### ISSUE TITLE
ability to customize default structure visibility

### ISSUE THREAD

**Issue by biscqit** [caeruleusDraconis/world-preview#40]
### Feature Request

title

**Comment by Kadeluxe** [comment]
Yes or at least remember latest toggled structures. Awesome mod btw!

**Comment by ASchneider-GitHub** [comment]
As an additional argument for adding this feature, using this mod on something like _AllTheMods 9_ causes the program to lag _massively_ as it tries to find a ton of different structures. If I wanted to just filter for Ancient Cities, I have to move the map and then wait for all the modded structures to get scanned for before I can disable everything except the ACs.
